app/selectors/post_list.js

```diff
@@ -58,9 +58,10 @@ export function makePreparePostIdsForPostList() {
 
                 // Push on a date header if the last post was on a different day than the current one
                 const postDate = new Date(post.create_at);
+                postDate.setHours(0, 0, 0, 0);
```

Comment:
What's all this for?

Reply:
the post date was being re-rendered cause the date was always different when a new post was received

Reply:
Oh, interesting. I hadn't thought of that before. That's a really good catch